packages/minimongo/modify.js

```diff
@@ -182,6 +182,40 @@ var NO_CREATE_MODIFIERS = {
 };
 
 var MODIFIERS = {
+  $currentDate: function(target, field, arg){
+    if(typeof arg === "object" && arg.hasOwnProperty("$type")){
+       if(arg.$type !== "date"){
+        throw MinimongoError("Minimongo does currently only support the date type in $currentDate modifiers");
+       }
+    }else if(arg !== true){
```

Comment:
Is there a style guide somewhere? The only reference I could find in the contributing.md refers to: https://guide.meteor.com/code-style.html#javascript which doesn't say a lot.